docs/EXTENDED-ARCHIVES.md

```diff
@@ -175,6 +179,9 @@ Here's an example of a real-world image manifest for Raspbian Jessie:
   "checksumType": "sha1",
   "path": "raspbian-jessie.img",
   "bmap": "raspbian-jessie.img.bmap",
+  "instructions": "raspbian-jessie.markdown",
+  "releaseNotes": "raspbian-jessie-changelog.txt",
+  "configurationSchema": "raspbian-jessie-schema.json",
```

Comment:
Would `reconfixSchema` be a better name? configurationSchema sounds a bit too generic IMHO.